libraries/monitor/src/index.js

```diff
@@ -211,8 +211,12 @@ class MonitorManager {
   }
 
   _unhandledRejectionHandler(reason, p) {
-    const err = 'Unhandled Rejection at: Promise ' + p + ' reason: ' + reason;
-    this.rootMonitor.reportError(err);
+    if (reason instanceof Error) {
```

Comment:
What if instead of this we just did

```js
_unhandledRejectionHandler(reason, p) {
  this.rootMonitor.reportError(reason);
  //... stuff about bailOnUnhandled here.
}
```

My reasoning is that `reportError` handles just about anything you throw into it including "error shaped objects" that aren't really `instanceof Error`. Also even [though the node docs say](https://nodejs.org/api/process.html#process_event_unhandledrejection) you can log `p`, this has always been a useless thing to write out afaict.

Reply:
yeah, sounds good.